src/core/mount.c

```diff
@@ -425,22 +426,6 @@ static bool mount_is_extrinsic(Unit *u) {
         return false;
 }
 
-static bool mount_point_is_credentials(Manager *manager, const char *path) {
-        const char *e;
-
-        assert(manager);
-        assert(path);
-
-        /* Returns true if this is a credentials mount. We don't want to generate mount units for them,
-         * since their lifetime is strictly bound to services. */
```

Comment:
Please keep the comment.

Reply:
This is now merged into `mount_setup_unit`. I don't think we should carry this in `exec-credential.c`...